NST/src/main/java/com/ebay/jsonpath/JPListOfIntegerCheck.java

```diff
@@ -190,7 +203,9 @@ public void processJsonPath(String jsonPath, SoftAssert softAssert, DocumentCont
       return;
     }
 
-    softAssert.assertNotNull(values, AssertMessageBuilder.build(jsonPath, "because the path does not exist."));
+    if (!isNullExpected()) {
```

Comment:
For collection types, is it worth adding another check empty as well for size ?

Reply:
Empty and size can both be evaluated by using the expected size check that already exists: https://github.com/eBay/NSTSuite/blob/main/NST/src/main/java/com/ebay/jsonpath/JPListOfIntegerCheck.java#L19